script/gn-check.js

```diff
@@ -0,0 +1,23 @@
+const sh = require('shelljs')
+const path = require('path')
+const fs = require('fs')
+
+const ELECTRON_DIR = path.resolve(__dirname, '..', '..')
```

Comment:
This looks like it would point to the directory `above` Electron. (Is this a copypaste error from `script/lib/utils.js`? :slightly_smiling_face:)

```suggestion
const { ELECTRON_DIR } = require('./lib/utils')
```

Reply:
@ckerr that's correct - this is intended to point to `src`, _not_ Electron. happy to change the name if that's clearer

Reply:
ahh of course, that makes sense. Maybe `ELECTRON_GN_DIR` to avoid confusion with the extant use of `ELECTRON_DIR` in unrelated `script/`s

Reply:
NVM, your new variable `GN_CHECK_DIR` is even better